packages/react-native-reanimated/src/ReanimatedModule/js-reanimated/JSReanimated.ts

```diff
@@ -290,7 +299,10 @@ export default class JSReanimated {
   }
 }
 
-enum Platform {
+// Lack of this export breaks TypeScript generation since
+// an enum transpiles into JavaScript code.
```

Comment:
Not sure why this change is necessary and why it didn't break TS generation prior to the changes.

Reply:
We talked about it offline.